### ISSUE TITLE
Server unresponsive every day at the same time

### ISSUE THREAD

**Issue by louisr0608** [Hermsi1337/docker-ark-server#65]
Hi,

I’m running into a strange problem where the server becomes unresponsive at almost exactly the same time every night. Everything in game becomes unresponsive, we can run around but not interact or see other people/dinos moving. Eventually everyone is booted off, but the server still shows up in the server list with the player count from when it froze up. This happens at almost exactly 12:35AM (just after midnight) server time every night, and the only fix I have is to restore a backup from 12:20AM and restart the server. After the restore/restart everything works great until the next night. This has happened consistently since I started the server, which was only a few days ago. 

I have backups running every 10 minutes, and if I restore the 12:30AM backup, the server won’t come up, as though the backup is corrupted.

Just as a side note, my server/container time is off from my local time by 4 hours (my mistake), so this actually happens at 8:35PM local time. 

This is mainly just a plea for assistance, looking for some insight on how to debug this problem. I figure this is likely something on my own system/setup causing this, but I was hoping there might be suggestions of something I can run in or out of the container to monitor what might be going on. I’ve checked for daily scheduled processes that might be causing problems, but can’t find anything. I have other Docker containers running that seem unaffected by whatever is causing this. 

Thanks in advance for any help or tips, and thank you for setting up and maintaining this container. My friends and I have been having a blast. 

Regards

**Comment by Hermsi1337** [comment]
This is a real strange behavior.. How did you set up the Backup? Via crontab? 

I'd suggest to set `DEBUG=true` and please share your configurations 😇

**Comment by louisr0608** [comment]
The crontab is run from the host machine, which executes 
docker exec ark_server arkmanager backup

Ive enabled debug and will grab the logs later today after I expect it to freeze up. 

I also updated the container when I turned debug on this morning..so maybe it will just sort itself out :)

**Comment by Hermsi1337** [comment]
⚠️ if you've upgraded to the current `latest`-image you have to adjust your backup-command as follows:

`docker exec ark_server arkmanager backup`

to

`docker exec -u steam ark_server arkmanager backup`

**Comment by louisr0608** [comment]
Thanks for the reminder -- I had forgotten to make that update. 

It seems something has resolved my issue... the only thing I changed was the Debug flag and the new container, but the server has been up since I restarted it two days ago without issue. The only thing I can think of is that the 'crash' wasn't really a crash, and is rather just a period of lag that boots people off and locks up the server temporarily. It may still be happening at the ~24 hour from restart mark, the difference now being that instead of having started/restarted the server around 8:30pm when I normally play, I restarted it much earlier in the day and just haven't been online the experience any issues.

Out of curiosity, I may restart the server around the time I play and see if this still happens the next day. Either way, I appreciate your input and time. If I do experiment with the restart time and come across anything, I'll follow up!

Regards
